src/lib/arch/aarch64/InstructionMetadata.cc

```diff
@@ -965,6 +1320,14 @@ void InstructionMetadata::revertAliasing() {
         }
         return;
       }
+      if (opcode == Opcode::AArch64_MUL_ZPmZ_B ||
+          opcode == Opcode::AArch64_MUL_ZPmZ_D ||
+          opcode == Opcode::AArch64_MUL_ZPmZ_H ||
+          opcode == Opcode::AArch64_MUL_ZPmZ_S) {
+        // Decode doesn't recognise instruction properly - akin to a self alias
```

Comment:
Could you explain this in a bit more detail? What do you mean by "Decode doesn't recognise instruction properly" for example

Reply:
Ill expand the comment too - but basically instead of a NYI exception it would print out a `Encountered illegal instruction exception` with `<unknown>` instead of the instruction just like with an alias. However from the opcode ID given and the corresponding instruction in the object dump are infact the same (in this case both z-reg multiplications)

Reply:
Removed comment after further discussion